app/webpacker/components/Persons/NewPersonForm/NewPersonForm.js

```diff
@@ -140,7 +137,7 @@ function NewPersonForm({
             name,
             dob,
             gender,
-            country_id: countryIdForIso2(countryIso2),
+            countryId: countryIdForIso2(countryIso2),
```

Comment:
Just curious, why did you change this from `snake_case` to `camelCase`? The backend usually expects snake case and I don't see any relevant changes to the backend in this PR...

Reply:
Uh this whole file looks like it might just be a mishandled rebase conflict. I didn't check carefully before pushing, since it's just a draft and I'm more looking for feedback on the UI rather than on the code, at this point.